process/constants.go

```diff
@@ -94,37 +94,9 @@ const RoundModulusTrigger = 5
 // RoundModulusTriggerWhenSyncIsStuck defines a round modulus on which a trigger for an action when sync is stuck will be released
 const RoundModulusTriggerWhenSyncIsStuck = 20
```

Comment:
should this also be extracted to a config? will it remain 20 also with 0.6s rounds?